bitnami/influxdb/README.md

```diff
@@ -304,7 +306,7 @@ There are K8s distribution, such as OpenShift, where you can dynamically define
 | `ingress.pathType`         | Ingress path type                                                                                                                | `ImplementationSpecific` |
 | `ingress.apiVersion`       | Force Ingress API version (automatically detected if not set)                                                                    | `""`                     |
 | `ingress.hostname`         | Default host for the ingress resource (evaluated as template)                                                                    | `influxdb.local`         |
-| `ingress.path`             | Ingress path*' in order to use this                                                                                              | `/`                      |
+| `ingress.path`             | Ingress path\*' in order to use this                                                                                             | `/`                      |
```

Comment:
This change is being inserted when I run the `readme-generator` utility; not sure if that's a problem.